modules/test/playwright/tests/frontend-js-components-web/translationManager.spec.ts

```diff
@@ -0,0 +1,525 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {clickAndExpectToBeHidden} from '../../utils/clickAndExpectToBeHidden';
+import {clickAndExpectToBeVisible} from '../../utils/clickAndExpectToBeVisible';
+import {jsComponentsSamplePageTest} from './fixtures/jsComponentsSamplePageTest';
+
+export const test = mergeTests(
+	dataApiHelpersTest,
+	apiHelpersTest,
+	featureFlagsTest({
+		'LPS-178052': {enabled: true},
+	}),
+	jsComponentsSamplePageTest,
+	isolatedSiteTest,
+	loginTest()
+);
+
+test.beforeEach(
+	'Setup site and JS Components Sample widget',
+	async ({apiHelpers, jsComponentsSamplePage, page, site}) => {
+		await test.step('Create a content site and the js components sample widget', async () => {
+			await jsComponentsSamplePage.setupJSComponentsSampleWidget({
+				apiHelpers,
+				site,
+			});
+		});
+
+		await test.step('Select Translation Manager tab', async () => {
+			await jsComponentsSamplePage.selectTab(
+				'Translation Manager',
+				page.getByRole('heading', {name: 'AUI Tag'})
+			);
+		});
+	}
+);
+
+test.describe('Translation Manager', () => {
+	test(
+		'Assert the localization dropdown is rendered alongside the localized input',
+		{tag: '@LPD-47235'},
+		async ({page}) => {
+			const localizedInputContainer = page
+				.locator('.input-localized.input-localized-input')
+				.first();
```

Comment:
Not a necessary change, but to make the locators more reliable and reduce some repetitiveness I'd suggest making this and other similar ones into a dedicated locator in `JSComponentsSamplePage.ts`.

You could also add a `data-qa-id` attribute in the `translation_manager.jsp` to make this locator more specific by using `page.getByTestId`